modules/living_spaces_activity/config/install/block.block.views_block__message_persistent.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - views.view.message
+  module:
+    - views
+  theme:
+    - living_spaces_barrio
```

Comment:
Theme is installed in the end so all blocks should be moved to profile config files as those installed after theme is ready.

Reply:
moved the block